WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController.m

```diff
@@ -364,6 +364,9 @@ - (void)viewWillAppear:(BOOL)animated
 - (void)viewDidAppear:(BOOL)animated
 {
     [super viewDidAppear:animated];
+    if ([self.tabBarController isKindOfClass:[WPTabBarController class]]) {
+        [(WPTabBarController *)self.tabBarController showCreateButton];
```

Comment:
Just a thought/question (does not need to be addressed now): if the `create button` only pertains to `BlogDetailsViewController`, should `WPTabBarController` still handle or be aware of it?

Reply:
> if the create button only pertains to BlogDetailsViewController, should WPTabBarController still handle or be aware of it?

It definitely could move into BlogDetailsViewController, but the button would still need to be added to the WPTabBarController's view. In my opinion, it seems like WPTabBarController should hold some kind of reference to it because of that.

The view setup code could be moved out of the TabBarController extension if that seems better, though.